src/app/data/DatabaseConnection.java

```diff
@@ -0,0 +1,36 @@
+package app.data;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+/**
+ * This Class deals with the connection to the database as well as any database controls.
+ *
+ * @author chad_olsen
+ * @since 2017/06/09.
+ */
+public class DatabaseConnection {
+
+    private static final String MYSQL_DRIVER = "com.mysql.jdbc.Driver"; // MySQL Driver name.
+    private static final String URL = "jdbc:mysql://localhost:3306/MediaCatalogue"; // MySQL Database URL for Media Catalogue database.
+    private static final String USER = "chad"; // Database username.
```

Comment:
Please make the username and password root. This is a global username and password for MySQL